XamarinCommunityToolkit/Behaviors/Validators/CharacterType.shared.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace Xamarin.CommunityToolkit.Behaviors
+{
+	[Flags]
+	public enum CharacterType
+	{
+		LowercaseLetter = 1,
+		UppercaseLetter = 2,
+		Letter = LowercaseLetter | UppercaseLetter,
+		Digit = 4,
+		LetterOrDigit = Letter | Digit,
+		WhiteSpace = 8,
+		Symbol = 16,
+		Any = LetterOrDigit | Symbol
+	}
```

Comment:
@jfversluis do you think we can add "EnglishLetter" as a constraint? Sometimes logins and passwords have such constraint

Reply:
Hm yeah that might be a good one. Although "English" is not the best term, probably something with [Latin](https://en.wikipedia.org/wiki/Latin_alphabet)? How do other frameworks call such a thing?

Reply:
For the rest looks good to me :)

Reply:
Frankly speaking, I don't know)
Perhaps **BasicLatinLetters**?

Reply:
Also, I can rename "LetterOrDigit" to "AlphaNumeric"

Reply:
I would go for something like `AlphaNumeric` I think that's better. If you change that, all the rest looks good :)

Reply:
@jfversluis updated